CONTRIBUTING.md

```diff
@@ -3,15 +3,15 @@
 ----
 
 <p align="center">
```

Comment:
Two things:

- Let's add "toc" class to this, so we can hide these on the website, no sense in having TOC links here AND on the side.

- Let's revert back to the anchor links:

![image](https://cloud.githubusercontent.com/assets/56288/23713211/c9e9e21a-03ea-11e7-8f3b-7bf1ebd7d5de.png)